tests/rest/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "lwm2m-rest-tests",
```

Comment:
Although this is not a separate npm package, please rename to "punica-rest-tests" to avoid package name collision.